integration_tests/metric_test.py

```diff
@@ -260,3 +261,49 @@ def test_qa_metrics(self):
             2,
             "almost equal",
         )
+
+    @unittest.skip(reason="dependency libs are not installed")
```

Comment:
Could we re-enable all skipped tests? In general, invalidated tests are not very useful to check consistency of the software.

If we really need to disable these tests, please add following things:

- which library is required to invoke this test
- why the library is not added to the dependency